.github/workflows/cd.yml

```diff
@@ -1,5 +1,5 @@
 name: CD
-
+run-name: CD ${{ github.run_id }} on ${{ github.ref_name }} with preid ${{ inputs.preid }}
```

Comment:
no need to add `${{ github.run_id }}` as it will be included automatically?
![image](https://user-images.githubusercontent.com/15644078/230870189-2e6b4192-e12a-4801-a4eb-5aa472ef2709.png)

Reply:
this is not run_id